stdlib/2and3/logging/__init__.pyi

```diff
@@ -18,6 +18,7 @@ else:
     _ExcInfoType = Union[None, bool, _SysExcInfoType]
 _ArgsType = Union[Tuple[Any, ...], Dict[str, Any]]
 _FilterType = Union['Filter', Callable[['LogRecord'], int]]
+_Level = Union[int, str, Text]
```

Comment:
Sure. I suppose it would no longer be redundant if we do decide to change what Text means, but it doesn't seem like there are any plans in that direction any more.